client/packages/core/src/Reactor.js

```diff
@@ -1295,8 +1332,8 @@ export default class Reactor {
 
     const handler = { ...opts, roomId, cb, prev: null };
 
+    this._rooms[roomId].isLoading = true;
```

Comment:
we would 'always' want to set this? i.e what if the user has already joined the room before?

Reply:
You're right.  This shouldn't be set here at all!